### ISSUE TITLE
Links sometimes open in a new tab

### ISSUE THREAD

**Issue by ccchapman** [spicywebau/craft-neo#433]
#### Description

Linking to an entry within Redactor can add undesired `target` and `rel` attributes under specific conditions. The issue cannot be reproduced when Neo is uninstalled or Neo's `main.js` file is blocked from loading.

Is there something in Neo's front-end or back-end code that could be causing this?

https://user-images.githubusercontent.com/42548502/110687016-e611c800-8194-11eb-87b5-213038e1d398.mp4

#### Steps to reproduce
1. Clean installation of Craft, Neo, and Redactor
2. Separate Craft's front-end and CP domains e.g. admin.craft.test and craft.test.
    - Use Craft's `baseCpUrl` configuration and set `cpTrigger` to `null`.
3. Create a Redactor field using the default configuration
4. Create a Neo field with a block for the Redactor field
5. Create two entries and insert a link from one to the other
    - No `target` or `rel` attribute appears
6. Save the entry
    - Notice `target` and `rel` appear on the link
7. Save the entry _again_

At this point, the `target` and `rel` attributes persist on the field and will output to the front-end.

<details>
<summary>
templates/index.twig
</summary>

```twig
{% for block in entry.contentBuilder.all() %}
  {{ block.editorStandard }}
{% endfor %}
```
</details>

**NOTES:**

You can also reproduce this issue if the Redactor field is not in the Neo field but they are both on the same entry type.

I suspect this was somewhat recently introduced. We have similar setups using earlier releases (Craft 3.5.17, Neo 2.8.16, same Redactor version) where the issue cannot be reproduced.

#### Other information
- Neo version: 2.9.2
- Craft version: 3.6.10
- Redactor version: 2.8.5

**Comment by ttempleton** [comment]
This does seem to have been recently introduced -- I've narrowed it down to the Craft 3.6.7 update so far.  Will keep looking into *exactly* what change caused it, to see whether it happened to activate a bug in Neo, or it's a Craft bug.

**Comment by ttempleton** [comment]
> Will keep looking into _exactly_ what change caused it, to see whether it happened to activate a bug in Neo, or it's a Craft bug.

Turns out it was a bit of both!  The occurrence of this on Redactor fields *not* on a Neo field does appear to be caused by a Craft bug, which I've reported at craftcms/cms#7686.  We'll still need to implement a fix for Neo fields specifically, which should be released in the next day.

**Comment by ttempleton** [comment]
Just released Neo 2.9.3 which should fix the Neo bug.
